source/styleguide/variables/colors.css

```diff
@@ -1,19 +1,24 @@
 /*
-The color palette for the project.
-http://cssnext.io/features/#color-function
+  The color palette for the project.
+  http://cssnext.io/features/#color-function
 
-background-color: var(--primary);
-color: var(--greyDark);
+  background-color: var(--primary);
+  color: var(--greyDark);
+
+  IMPORTANT:
+  In order for the styleguide colors to not be included by the SgColorSwatch,
+  please prefix all variables inside this file with '--sg-color'.
 */
+
 :root {
```

Comment:
Until you restart dev, any renaming of colors in here is not replacing colors in SgColorSwatch_variables.json, but instead only adding colors to it. This leaves you in a state where you have colors in SgColorSwatch__variables.json that don't actually exsist in project anymore.

Reply:
Styleguide colors should not be compiled into the color swatch